### DISCUSSION TITLE
The community container "pi-hole" was not found!  | You've set AIO_COMMUNITY_CONTAINERS but at least one container was not found.

### DISCUSSION THREAD

**Discussion by manishmaharjan** [nextcloud/all-in-one#4171] [Questions] [answered]
Hi,

Using Nextcloud AIO v7.11.2. I have tried with both beta and latest version, both had this following error and pi-hole failed to start properly. 
https://github.com/nextcloud/all-in-one/tree/main/community-containers/pi-hole

This was tried in macOS and there is nothing running in port 53 and no any local dns server running either. 

I tested both ways with and without quotation, didn't make any difference. 
      - AIO_COMMUNITY_CONTAINERS=pi-hole
      - AIO_COMMUNITY_CONTAINERS="pi-hole"
      
```dockerfile
➜  docker-compose up
WARN[0000] volume "nextcloud_aio_mastercontainer" already exists but was not created by Docker Compose. Use `external: true` to use an existing volume
[+] Running 2/2
 ✔ Network nextcloud_default                Created                                                                                                                             0.1s
 ✔ Container nextcloud-aio-mastercontainer  Created                                                                                                                             0.0s
Attaching to nextcloud-aio-mastercontainer
nextcloud-aio-mastercontainer  | Trying to fix docker.sock permissions internally...
nextcloud-aio-mastercontainer  | Adding internal www-data to group root
nextcloud-aio-mastercontainer  | WARNING: daemon is not using the default seccomp profile
nextcloud-aio-mastercontainer  | The community container "pi-hole" was not found!
nextcloud-aio-mastercontainer  | You've set AIO_COMMUNITY_CONTAINERS but at least one container was not found.
nextcloud-aio-mastercontainer  | It is set to '"pi-hole"'.
nextcloud-aio-mastercontainer exited with code 0
```

docker-compose file contents 
```dockerfile

services:
  nextcloud-aio-mastercontainer:
    image: nextcloud/all-in-one:latest
    init: true
    restart: always
    container_name: nextcloud-aio-mastercontainer
    volumes:
      - nextcloud_aio_mastercontainer:/mnt/docker-aio-config
      - /var/run/docker.sock.raw:/var/run/docker.sock:ro
    ports:
      - 80:80
      - 8080:8080
      - 8443:8443
    environment:
      - AIO_COMMUNITY_CONTAINERS=pi-hole

volumes:
  nextcloud_aio_mastercontainer:
    name: nextcloud_aio_mastercontainer
``` 

```
➜   docker images
REPOSITORY                  TAG       IMAGE ID       CREATED      SIZE
nextcloud/all-in-one        beta      80c8e658b3aa   2 days ago   229MB
nextcloud/all-in-one        latest    ff7f6be15c6c   5 days ago   229MB
nextcloud/aio-domaincheck   latest    772ee4c0f9f9   5 days ago   13.6MB
```

I also don't see any pi-hole images pulled either. 
Any help would be appreciated. 
Thank you!

**Comment by szaimen** [comment]
> ```dockerfile
>     environment:
>       - AIO_COMMUNITY_CONTAINERS=pi-hole
> ```

This looks good. Did you aleady open the aio interface and checked if the container is shown there?

**Reply by manishmaharjan** [reply]
Yup, there is no sign of that container in aio interface or docker process. There was also others which failed to start. Only thing that worked for me was plex. 
Has this pi-hole been tested out with clean install recently?

**Reply by szaimen** [reply]
> Yup, there is no sign of that container in aio interface or docker process. There was also others which failed to start. Only thing that worked for me was plex.

So did you use this then?
`- AIO_COMMUNITY_CONTAINERS=pi-hole plex`
And after running docker compose down and docker compose up and opening the aio interface, it did not show both containers?

**Reply by manishmaharjan** [reply]
yup, only plex worked, but not pi-hole.

**Reply by manishmaharjan** [reply]
I checked and still same issue for me...

**Reply by szaimen** [reply, marked as answer]
Please run `sudo docker compose up` with the correct compose file, afterwards please post the output of `sudo docker inspect nextcloud-aio-mastercontainer` here and additionally a screenshot of what you see in the AIO interface. Thanks!
